qutebrowser/config/configcommands.py

```diff
@@ -165,17 +167,28 @@ def bind(self, win_id: str, key: str = None, command: str = None, *,
 
         seq = self._parse_key(key)
 
+        # Since we don't split args, this argument is left unparsed, which
+        # means quotes are left behind.
+        if pattern is not None:
+            pattern = pattern.strip("\"'")
```

Comment:
I found a couple functions to do more complicated parsing, but I think in the end, we only just want this result. Is there a cleaner way to do this (making `:bind --pattern="*.a.b"` work?)